engine/bindings/python/interface.cpp

```diff
@@ -68,6 +55,101 @@ static PyObject *use_db(PyObject *self, PyObject *args)
   db_name = namePtr;
   return PyLong_FromLong(0);
 }
-static PyObject *create_table(PyObject *self, PyObject *args) {}
+static PyObject *create_table(PyObject *self, PyObject *args)
```

Comment:
does this mean we will need reimplement the interface and keep the native C++ and python binding implementation in sync from now on?

Reply:
what do you mean by "in sync"? as long as all required arguments are passed to the python interface, the C++ interface can be different.